tests/foreman/api/test_parameters.py

```diff
@@ -14,44 +14,84 @@
 import pytest
 
 
+@pytest.fixture
+def param_name():
```

Comment:
I think it would be better to just use parametrization for the test and generate the name and value there. You can pass those variables to your other fixtures using `request`.

Reply:
To give you an example see how `repo_options` works in https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_repository.py#L36